pkg/server/middleware.go

```diff
@@ -43,6 +43,23 @@ func authMiddlewareWithUser(authenticator auth.Authenticator, csrfVerifier *csrf
 	)
 }
 
+// authMiddlewareWithoutToken authenticates the user but doesn't set the Authorization header.
+// This allows handlers to use internalProxiedK8SRT transport with console service account token.
+func authMiddlewareWithoutToken(authenticator auth.Authenticator, csrfVerifier *csrfverifier.CSRFVerifier, h http.HandlerFunc) http.HandlerFunc {
```

Comment:
Do we still need this now that we switched to client-go and are no longer using the proxy?

Reply:
@spadgett  Yes, we still need it, as only authenticated console users should have access to this endpoint. This is used to verify they're logged into the console